indexer.py

```diff
@@ -17,6 +18,8 @@ def index(document, importer, data_accessor):
     # then over each word in each line. When the `line_index` of the current
     # word matches the last word index of the current line, then that word must
     # be at the end of a line.
+    stop_words = set(stopwords.words('english'))
```

Comment:
Why do you need to convert this to a set? Also, can this be defined in main code after imports, so you don't have to create the variable each time you call index?